src/alarm_trap_sender.cpp

```diff
@@ -143,3 +166,112 @@ void AlarmTrapSender::send_trap(const AlarmTableDef& alarm_table_def)
 
   snmp_reset_var_buffers(&var_trap);
 }
+
+void AlarmTrapSender::send_enterprise_trap(const AlarmTableDef& alarm_table_def)
+{
+  static const unsigned int severity_to_state[] = {2, 1, 2, 6, 5, 4, 3};
+  static const oid snmp_trap_oid[] = {1,3,6,1,6,3,1,1,4,1,0};
+  static const oid trap_type_oid[] = {1,3,6,1,4,1,9,19444,12,2,0,1};
```

Comment:
My bad on a previous comment - the "9" before "19444" shouldn't be there. (Searching 'SNMPv2-SMI enterprises' turns up http://www.alvestrand.no/objectid/submissions/1.3.6.1.4.1.9.html, but it looks like http://www.alvestrand.no/objectid/1.3.6.1.4.1.html is the correct one.)